### PR TITLE
Rules

### PR DESCRIPTION
**Pull request by amantulsyan35**
Rules page for the quiz application

### CODE REVIEW

File: Question/question.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  margin: 2rem 7rem 2rem 7rem;
+  padding: 1rem;
+  height: auto;
+}
+
+.question-quiz-heading {
+  box-shadow: 0 2px 2px 2px rgba(0, 0, 0, 0.1);
+  padding: 1rem;
+  margin-bottom: 1rem;
+  display: flex;
```
Comment (parvez-rashid) [inline Question/question.css:12]: You can use a utility class for `display: flex`. It'll be easier to use.

File: Result/result.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  margin: 2rem 7rem 2rem 7rem;
+  padding: 1rem;
+  height: auto;
+}
+
+.result-heading {
+  border: 1px solid rgb(0, 0, 0, 0.2);
+  padding: 1rem;
+  margin-bottom: 1rem;
+  display: flex;
```
Comment (parvez-rashid) [inline Result/result.css:14]: You can use a utility class for `display: flex`. It'll be easier to use.

File: Result/result.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  justify-content: center;
+}
+
+.result-details {
+  margin-bottom: 1rem;
+  box-shadow: 0 2px 2px 2px rgba(0, 0, 0, 0.1);
+  padding: 1rem;
+  display: flex;
```
Comment (parvez-rashid) [inline Result/result.css:22]: You can use a utility class for `display: flex`. It'll be easier to use.

File: Result/result.css [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+.result-question-container {
+  border: 1px solid rgb(0, 0, 0, 0.2);
+  padding: 1rem;
+}
+
+.result-question {
+  border: 1px solid rgb(0, 0, 0, 0.1);
+  padding: 1rem;
+  display: flex;
```
Comment (parvez-rashid) [inline Result/result.css:34]: You can use a utility class for `display: flex`. It'll be easier to use.

**Review by parvez-rashid — APPROVED** [review]
UI Review: The same thing as with results page, borders are looking crowded. Again, just my perspective.. Have similar suggestion for code as well, have a look below. Apart from that, looks well-structured and consistent, can be merged.

**Review by kaustubh-jsr — COMMENTED** [review]
The overall UI is good, but a small suggestion, you can use a non-default font, to beautify it. The rest of the code looks good to merge.

**Pull request merged**